### PR TITLE
config: update pluto's peerID

### PR DESCRIPTION
**Pull request by lgierth**
We lost the private key of pluto while rebuilding that host a while ago

### CODE REVIEW

File: repo/config/bootstrap_peers.go [hunk 1]
```diff
@@ -15,14 +15,14 @@ import (
...
-	"/ip4/104.236.176.52/tcp/4001/ipfs/QmSoLnSGccFuZQJzRadHn95W2CrSFmZuTdDWP8HXaHca9z",  // neptune (to be neptune.i.ipfs.io)
-	"/ip4/104.236.179.241/tcp/4001/ipfs/QmSoLpPVmHKQ4XTPdz8tjDFgdeRFkpV8JgYq8JVJ69RrZm", // pluto (to be pluto.i.ipfs.io)
-	"/ip4/162.243.248.213/tcp/4001/ipfs/QmSoLueR4xBeUbY9WZ9xGUUxunbKWcrNFTDAadQJmocnWm", // uranus (to be uranus.i.ipfs.io)
-	"/ip4/128.199.219.111/tcp/4001/ipfs/QmSoLSafTMBsPKadTEgaXctDQVcqN88CNLHXMkTNwMKPnu", // saturn (to be saturn.i.ipfs.io)
-	"/ip4/104.236.76.40/tcp/4001/ipfs/QmSoLV4Bbm51jM9C4gDYZQ9Cy3U6aXMJDAbzgu2fzaDs64",   // venus (to be venus.i.ipfs.io)
-	"/ip4/178.62.158.247/tcp/4001/ipfs/QmSoLer265NRgSp2LA3dPaeykiS1J6DifTC88f5uVQKNAd",  // earth (to be earth.i.ipfs.io)
-	"/ip4/178.62.61.185/tcp/4001/ipfs/QmSoLMeWqB7YGVLJN3pNLQpmmEk35v6wYtsMGLzSr5QBU3",   // mercury (to be mercury.i.ipfs.io)
-	"/ip4/104.236.151.122/tcp/4001/ipfs/QmSoLju6m7xTh3DuokvT3886QRYqxAzb1kShaanJgW36yx", // jupiter (to be jupiter.i.ipfs.io)
+	"/ip4/104.236.176.52/tcp/4001/ipfs/QmSoLnSGccFuZQJzRadHn95W2CrSFmZuTdDWP8HXaHca9z",  // neptune.i.ipfs.io
+	"/ip4/104.236.179.241/tcp/4001/ipfs/QmWUMtbNSEsgYKcaAeT5XwyojSYQVnhhQdMHeaCkaxbMxA", // pluto.i.ipfs.io
```
Comment (jbenet) [inline repo/config/bootstrap_peers.go]: @whyrusleeping do you want to mine for a `SoL[pP]`?

Comment (lgierth) [inline repo/config/bootstrap_peers.go]: Oh I hadn't even noticed that pattern. Neat!

Comment (whyrusleeping) [inline repo/config/bootstrap_peers.go]: @jbenet sure, i'll get one done

**Comment by whyrusleeping** [comment]
@lgierth which public key should I use to encrypt and send plutos new key to you?

**Comment by lgierth** [comment]
@whyrusleeping http://zimmermann.mayfirst.org/pks/lookup?op=get&fingerprint=on&search=0xF42CE81D7E86809F

**Comment by whyrusleeping** [comment]
@lgierth plutos key is here (signed and encrypted): QmaWLe7PaTK2V1teN6UL2ihJbDJLG4Hx4Zd23Di4qg7JZo

**Comment by lgierth** [comment]
Thanks, here we go -- it's already running with the new key

**Comment by lgierth** [comment]
Fixing the failing dht test

**Comment by lgierth** [comment]
Mh, it's a timeout in `TestConnectCollision`, and it's failing on master too

**Comment by lgierth** [comment]
Well, it's flapping -- maybe unrelated to the changed private key

**Comment by lgierth** [comment]
Rebased onto master, it's green now

**Pull request merged**